core/example/longrange/direct.h

```diff
@@ -0,0 +1,30 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef TDS_TDS_INCLUDED
+#define TDS_TDS_INCLUDED
+
+#include <iostream>
+#include "definitions.h"
+
+class TDS
```

Comment:
Add some comments here - what does `TDS` mean?

Reply:
Originally Rene named the solver classes with a prefix of "T" (I'm not sure what that indicates) - TDS meant Direct Sum, TEwald is Ewald Sum.
I'm thinking something more clear would look like (Direct, Ewald, SPME)?